deployment/ib-kubernetes-deploy.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
   namespace: kube-system
   annotations:
     kubernetes.io/description: |
-      This daemonset launches the ib-kubernetes daemon for infiniband cni.
+      This deployment launches the ib-kubernetes daemon for infiniband CNI.
```

Comment:
a/infiniband/InfiniBand

Reply:
we also should mark it as critical [1]
[1] -https://github.com/ovn-org/ovn-kubernetes/blob/master/dist/templates/ovnkube-master.yaml.j2#L33-L34